starjudge2/main.py

```diff
@@ -7,9 +7,9 @@
 
         
 POINTS = 50
-A = 25
-B = 50
-C = -abs(25)
+A = 20

```

Comment:
Can you think of a more descriptive name for this variable?

Reply:
Let's call it 

`good_action
meh_action
bad_action
`